services/home/src/JoinStudy/components/JoinStudyBox/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button, Image, Tag, Text } from '@depromeet/toks-components';
+import { Flex, Spacing, width100 } from '@toss/emotion-utils';
+
+import { StudyInfo } from 'JoinStudy/components/StudyInfo';
+import { StudyTitle } from 'JoinStudy/components/StudyTitle';
+
+import { Wrapper } from './style';
+
+export function JoinStudyBox() {
+  // mock data
+  const ourStudyDescription =
+    '설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한';
+  const personnelDescription = '5-7명을 계획하고 있어요.';
+  const startDate = '2022. 10. 13';
+  const doneDate = '2022. 12. 03';
+  const tags = ['Java', 'Javascript', 'React'];
+  return (
+    <Wrapper>
+      <div>
+        <StudyTitle />
+        {/* tag margin 위아래 10 고려하여 18->8 */}
+        <Spacing size={8} />
+        <Tag.Row>
+          {tags?.map(tag => (
+            <Tag key={tag}>{tag}</Tag>
+          ))}
+        </Tag.Row>
+      </div>
+      <Flex direction="column" css={{ gap: '32px' }}>
+        <StudyInfo
+          leftAddon={
+            <Image
+              width={16}
+              height={22}
+              src="https://toks-web-assets.s3.amazonaws.com/studyinfo-icon.svg"
+              alt="StudyInfo 아이콘"
+            />
+          }
+          title={'우리 스터디는'}
```

Comment:
```suggestion
          title="우리 스터디는"
```
nit. 사소하긴한데 string은 그냥 입력해도 괜찮을 것 같아요~?!

Reply:
넵!!! 🤩